### PR TITLE
Docker images for Github Actions Agent

### PR DESCRIPTION
**Pull request by rakku-ms**
Updating docker images so that it can be used for running Github Actions agents.

User will create a vm, add DockerExtension and then can configure agent runner within the container 

```
az vm create --resource-group "test" --name "testVM" --image "UbuntuLTS" --admin-username "demouser" --admin-password "demouser@111" --location "local"
az vm extension set -n DockerExtension --publisher Microsoft.Azure.Extensions --vm-name testVM -g test
```

### CODE REVIEW

**Comment by rakku-ms** [comment]
@bganapa and @sarathys should we remove install and test script files from the generated docker images?
```
PS /> ls
anaconda-post.log  Download-ActionsAgent.ps1  Install-AzCLICentOS.ps1           lib                    media                            opt   run   sys                     usr
bin                etc                        Install-AzureStackPowerShell.ps1  lib64                  mnt                              proc  sbin  Test-AzsPowershell.ps1  var     
dev                home                       Install-AzureStackTools.ps1       Login-Environment.ps1  New-AzureStackTestResources.ps1  root  srv   tmp
```

File: docker/Dockerfile-centos-7 [hunk 1]
```diff
@@ -45,4 +45,10 @@ RUN Install-Module -Name Microsoft.AzureStack.ReadinessChecker -RequiredVersion
 # create AzureRmContextSettings.json before it was generated
 COPY ${CONFIG}/${AZURERM_CONTEXT_SETTINGS} ${AZURE}/${AZURERM_CONTEXT_SETTINGS}
```
Comment (rakku-ms) [inline docker/Dockerfile-centos-7:46]: @bganapa do we need this AzureRMContextSetting for Az too?

Comment (bganapa) [reply docker/Dockerfile-centos-7:46]: Az settings are stored in a file still called as AzureRM*

Comment (bganapa) [reply docker/Dockerfile-centos-7:46]: @TheOnlyWei IIRC, we did this as a workaround, do we still need to copy?

Comment (TheOnlyWei) [reply docker/Dockerfile-centos-7]: @bganapa I am not sure, I never don't remember using AzureRM or copying context files when I worked on the container. I used Az.

**Pull request closed without merging**